.clang-tidy

```diff
@@ -0,0 +1,15 @@
+# See https://clang.llvm.org/extra/clang-tidy/checks/list.html
+
+Checks: '
+-*,
+clang-diagnostic-*,
```

Comment:
I don't see `clang-diagnostic-*` in the list of clang-tidy operations.

Reply:
Right, removed this from the list of checks